runtime/jcl/common/jdk_internal_vm_Continuation.cpp

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "jni.h"
+#include "jcl.h"
+#include "jclprots.h"
+#include "ut_j9jcl.h"
+
+extern "C" {
+
+#if JAVA_SPEC_VERSION >= 19
```

Comment:
Is this directive required since the file is only included for JDK19+?

Reply:
Yes.

Reply:
In that case, `#if JAVA_SPEC_VERSION >= 19` is not required then.

Reply:
Removed from `jdk_internal_vm_Continuation.cpp` but kept in `runtime/oti/jclprots.h` since it is shared from all Java versions.